google_shopping_functions.js

```diff
@@ -1,30 +1,78 @@
-//define getItemsCount function
-//accepts full item data
-//returns the length of the items array
+
 var getItemsCount = function(itemData){
+	for(items in itemsData){
```

Comment:
We don't need this for loop here, because we aren't looping through anything. You can take this part out here.